backend/typescript/services/implementations/EntityService.ts

```diff
@@ -0,0 +1,94 @@
+import MgEntity, { Entity } from "../../models/entity.mgmodel";
+import { IEntityService, EntityRequestDTO, EntityResponseDTO } from "../interfaces/IEntityService";
+
+
+export class EntityService implements IEntityService {
+
+    async getEntity(id: string): Promise<EntityResponseDTO> {
+        let entity: Entity | null;
+        entity = await MgEntity.findById(id);
```

Comment:
Wasn't there a try-catch around this before? I think it's still necessary because `findById` can throw if you pass in a malformed id